src/github/pagination.ts

```diff
@@ -17,12 +17,12 @@ async function paginate (octokit: GitHubAPI, responsePromise: any, callback = de
 
   let response = await responsePromise
 
-  collection = collection.concat(await callback(response, done))
```

Comment:
Are you sure that this is not a potentially breaking change? People might return a promise in the callback function. By removing the await, the `.paginate()` method would now return an array of promises instead of the resolved values.

I don’t think this was intended and I would prefer to remove it, but I wonder if someone is using this today?

Reply:
@gr2m Well the new/incorrect TS callback definition was introduced lately. I really doubt that anybody is using it. Only if their using it in plain JS. Anyway you should decide if you want to keep it and if so the definition and implementation should be updated accordingly...But even in docs the callback does not return `Promise`.

Reply:
> But even in docs the callback does not return `Promise`.

That’s a good point, I’m okay with removing it as it has not been documented as such. If it breaks someone code then they used an internal behavior that we can consider to have been an actual bug, and we can help them to migrate to another solution.

So all good from my side